### ISSUE TITLE
Sharing my interesting results with rectangle renderer

### ISSUE THREAD

**Issue by seungwonpark** [jiupinjia/stylized-neural-painting#1]
Hi, thanks for open-sourcing the great work, and I just wanted to share my own results.
(I think it'll be great if there's a place for users to post their own results, which will be some kind of "open gallery")

The input was my profile picture (drawn with LuaTeX+TikZ before, which imitated the JOI logo - https://git.io/vQwmS)

![veydpz_input](https://user-images.githubusercontent.com/13394294/99347592-cc453000-28da-11eb-88c0-787fcb7ca3a8.png)

The progressive output was interesting; the network preferred to draw a tilted rectangle, rather than using an axis-aligned rectangle (can you think of a reason behind this?):
![image](https://user-images.githubusercontent.com/13394294/99347673-fb5ba180-28da-11eb-9f61-c9279a796ea3.png)

The final result looked like this:
![veydpz_rendered_stroke_0495](https://user-images.githubusercontent.com/13394294/99347704-0adaea80-28db-11eb-84b4-84248f8c887e.png)

**Comment by jiupinjia** [comment]
Hi, thanks for your result! To generate axis-aligned strokes, we need to add a constraint on the orientation of the stroke parameters. You can try: pt.x_ctt.data[:, :, -1] = torch.clamp(pt.x_ctt.data[:, :, -1], 0, 0)

I have uploaded a demo script on this feature (see demo_8bitart.py). I have the following results on my side. You can try it and tell me whether it works fine. Thanks!
![test_rendered_stroke_0005](https://user-images.githubusercontent.com/51430557/99348950-c57dd500-2868-11eb-972c-0280b2aa7ed9.png)
![test_rendered_stroke_0495](https://user-images.githubusercontent.com/51430557/99348965-cc0c4c80-2868-11eb-9d77-a5c94e81aff2.png)

**Comment by seungwonpark** [comment]
Great idea! It's working really well on my side, too. Thanks a lot.
